### PR TITLE
Add crashlytics to sample app

### PR DESCRIPTION
**Pull request by vincent-karuri**

### CODE REVIEW

File: sample/src/main/java/io/ona/kujaku/sample/activities/MainActivity.java [hunk 1]
```diff
@@ -100,6 +102,8 @@ public void onClick(View v) {
         });
 
         setTitle(R.string.main_activity_title);
+        // initialize fabric
+        Fabric.with(this, new Crashlytics());
```
[the remark this answers is not in the document]
Comment (vincent-karuri) [reply sample/src/main/java/io/ona/kujaku/sample/activities/MainActivity.java]: Moved this to application class.

File: build.gradle [hunk 2]
```diff
@@ -2,23 +2,27 @@
 
 buildscript {
     repositories {
+        google()
         jcenter()
         maven {
```
Comment (ekigamba) [inline build.gradle]: Remove this. It's a duplicate

File: build.gradle [hunk 3]
```diff
@@ -2,23 +2,27 @@
...
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
     }
 }
 
 allprojects {
     repositories {
+        google()
         jcenter()
         maven {
```
Comment (ekigamba) [inline build.gradle]: REmove this also

File: sample/build.gradle [hunk 4]
```diff
@@ -107,8 +109,15 @@ dependencies {
 
     implementation 'com.github.GrenderG:Toasty:1.3.0'
 
+    implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.0'
```
Comment (ekigamba) [inline sample/build.gradle]: Why add this?

Comment (vincent-karuri) [reply sample/build.gradle]: Removed.

File: build.gradle (hunk 2 above)
Comment (ekigamba) [reply build.gradle]: I think [this line](#https://github.com/onaio/kujaku/blob/8ca8b8d0602c821439144858326af3ff335be7d2/build.gradle#L7) is a duplicate of `google()`

Remove it, everything should work ok

File: build.gradle (hunk 3 above)
Comment (ekigamba) [reply build.gradle]: I think [this line](https://github.com/onaio/kujaku/blob/bec90b958819129b10fb4ac6d552605f8cae12cd/build.gradle#L27) is a duplicate of `google()`

Remove it, everything should work ok

Comment (vincent-karuri) [reply build.gradle]: This change does not work on my machine.

Comment (vincent-karuri) [reply build.gradle]: Again, we discussed this.

Comment (vincent-karuri) [reply build.gradle]: If there is another solution, please share it.

**Comment by vincent-karuri** [comment]
Updated with master.

**Pull request merged**